src/axom/sidre/core/View.cpp

```diff
@@ -1054,44 +1037,45 @@ bool View::isApplyValid() const
 {
   bool rv = false;
 
-  if ( !isDescribed() )
+  if(!isDescribed())
   {
-    SLIC_CHECK_MSG(false,
-                   SIDRE_VIEW_LOG_PREPEND
-                   << "Apply is not valid. View does not have a description.");
+    SLIC_CHECK_MSG(
+      false,
+      SIDRE_VIEW_LOG_PREPEND
+        << "Apply is not valid. View does not have a description.");
     return rv;
   }
 
-  switch (m_state)
+  switch(m_state)
   {
   case EMPTY:
   case STRING:
   case SCALAR:
     SLIC_CHECK_MSG(false,
                    SIDRE_VIEW_LOG_PREPEND
-                   << "Apply is not valid for View with state '"
-                   << getStateStringName(m_state) << "'.'");
+                     << "Apply is not valid for View with state '"
+                     << getStateStringName(m_state) << "'.'");
     break;
   case EXTERNAL:
-    SLIC_ASSERT ( m_external_ptr != nullptr );
+    SLIC_ASSERT(m_external_ptr != nullptr);
     rv = isDescribed();
     break;
   case BUFFER:
-    rv = 0 <= getTotalBytes() &&
-         getTotalBytes() <= m_data_buffer->getTotalBytes();;
-    SLIC_CHECK_MSG(0 <= getTotalBytes(),
-                   SIDRE_VIEW_LOG_PREPEND
-                   << "Apply is not valid on data with zero length." );
+    rv =
+      0 <= getTotalBytes() && getTotalBytes() <= m_data_buffer->getTotalBytes();
+    ;
```

Comment:
?

Reply:
Thanks! I removed the extraneous semicolon.